integration-tests/smoke/auto_ocr_test.go

```diff
@@ -0,0 +1,181 @@
+package smoke
+
+import (
+	"context"
+	"math/big"
+	"strconv"
+
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	eth "github.com/smartcontractkit/chainlink-env/pkg/helm/ethereum"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver"
+	mockservercfg "github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver-cfg"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/contracts/ethereum"
+	"github.com/smartcontractkit/chainlink-testing-framework/utils"
+
+	networks "github.com/smartcontractkit/chainlink/integration-tests"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions"
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+)
+
+var _ = Describe("Automation OCR Suite @keeper", func() {
```

Comment:
Should this test exist in the keeper_test.go since it is a keeper test?

Reply:
first file has all our non-ocr smoke tests and is already near 1k lines so figures this was a good logical point to break it into its own file.

Reply:
and allows for an easy way to just run the legacy tests or just new ocr2 tests

Reply:
I would say label it @auto